.travis.yml

```diff
@@ -34,9 +54,10 @@ before_script:
       echo yes | pecl install apcu-5.1.6
     fi
   - |
-    if [[ $TRAVIS_PHP_VERSION != 'hhvm' ]]; then
+    if [[ $TRAVIS_PHP_VERSION != 'hhv*' ]]; then
```

Comment:
This is the format I use in my repos and those that I contribute to. By using `hhv*`  it covers `hhvm`, `hhvm-nightly` and LTS specified versions like `hhvm-3.12`

Although I did forget to remove the tick marks which does cause a problem.